fdbclient/NativeAPI.actor.cpp

```diff
@@ -1555,7 +1557,7 @@ DatabaseContext::DatabaseContext(Reference<AsyncVar<Reference<IClusterConnection
 		    SpecialKeySpace::MODULE::GLOBALCONFIG,
 		    SpecialKeySpace::IMPLTYPE::READWRITE,
 		    std::make_unique<GlobalConfigImpl>(SpecialKeySpace::getModuleRange(SpecialKeySpace::MODULE::GLOBALCONFIG)));
-		registerSpecialKeysImpl(
+		registerSpecialKeySpaceModule(
```

Comment:
Was this intentional?

Reply:
No Seems like bad merge. Reverting this change.